iosxr_iso2vbox.py

```diff
@@ -368,9 +366,6 @@ def main(argv):
         "box build with ova export, verbose and upload to artifactory: iosxr-xrv64-vbox/iosxr_iso2vbox.py iosxrv-fullk9-x64.iso -o -v -a 'New Image'\n")
     parser.add_argument('ISO_FILE',
                         help='local ISO filename or remote URI ISO filename...')
-    parser.add_argument('-a', '--artifactory', nargs='?', metavar="'New box reason'",
```

Comment:
It looks like you should update the "Full help" section in the README to reflect the removal of this parameter.

Do you have any documentation for iosxr_store_box.py? Should you?